<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1458</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des chercheurs</string>
  </property>
  <property name="toolTipDuration">
   <number>-4</number>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{border-radius:13px;
margin-top :5px;
background-color:gris doha ;
border:2px solid white; }
QPushButton:hover{color:white ; 
background-color: gris doha;
border:2px solid white; }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="ModifierChercheur">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2120</width>
      <height>751</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter Chercheur</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>90</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>130</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>110</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>200</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Numéro de téléphone</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajout_pushButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>190</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../C++/Projet/icons8-ajouter-administrateur-50.png</normaloff>../C++/Projet/icons8-ajouter-administrateur-50.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_reset">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>190</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../C++/Projet/icons8-annuler-80.png</normaloff>../C++/Projet/icons8-annuler-80.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>160</width>
        <height>160</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_chrchreur_lineEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom_chercheur">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_email_chercheur">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>390</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_telephone_chercheur">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>470</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_age_chercheur">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>220</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>90</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cin_chercheur">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-180</y>
        <width>1591</width>
        <height>1060</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-radius:13px;
margin-top :5px;
background-color:gris doha ;
border:2px solid white; }
QPushButton:hover{color:white ; 
background-color: gris doha;
border:2px solid white; }</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../C++/backgroundDesktop.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>-10</y>
        <width>781</width>
        <height>711</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../C++/backgroundDesktop.png</pixmap>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>90</y>
        <width>530</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Supprimer ou modifier un chercheur</string>
      </property>
      <widget class="QPushButton" name="pushButton_Supprimer">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>170</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../C++/Projet/supprimer.png</normaloff>../C++/Projet/supprimer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cin_modifierOuSupprimer">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>230</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Modifier">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>170</width>
         <height>70</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../C++/Projet/modifier-le-fichier.png</normaloff>../C++/Projet/modifier-le-fichier.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>300</y>
        <width>970</width>
        <height>280</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: medium solid #000000;
width: 50%;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>10</y>
        <width>260</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../C++/Projet/chercher.png</normaloff>../C++/Projet/chercher.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>460</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>560</y>
        <width>260</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../C++/Projet/icons8-trier-les-réponses-64.png</normaloff>../C++/Projet/icons8-trier-les-réponses-64.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>560</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ordre Croissant</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>610</y>
        <width>140</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ordre Decroissant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Export">
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>590</y>
        <width>160</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Export PDF</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../C++/Projet/icons8-exporter-30.png</normaloff>../C++/Projet/icons8-exporter-30.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>180</width>
        <height>180</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Statistique">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>590</y>
        <width>190</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../C++/Projet/icons8-graphique-combiné-50.png</normaloff>../C++/Projet/icons8-graphique-combiné-50.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Excel">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>590</y>
        <width>140</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Excel</string>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>label_6</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>Ajout_pushButton</zorder>
     <zorder>pushButton_reset</zorder>
     <zorder>nom_chrchreur_lineEdit</zorder>
     <zorder>lineEdit_prenom_chercheur</zorder>
     <zorder>lineEdit_email_chercheur</zorder>
     <zorder>lineEdit_telephone_chercheur</zorder>
     <zorder>spinBox_age_chercheur</zorder>
     <zorder>label_16</zorder>
     <zorder>lineEdit_cin_chercheur</zorder>
     <zorder>groupBox</zorder>
     <zorder>tableView</zorder>
     <zorder>pushButton_19</zorder>
     <zorder>lineEdit_recherche</zorder>
     <zorder>pushButton_20</zorder>
     <zorder>checkBox</zorder>
     <zorder>checkBox_2</zorder>
     <zorder>pushButton_Export</zorder>
     <zorder>pushButton_Statistique</zorder>
     <zorder>pushButton_Excel</zorder>
    </widget>
    <widget class="QWidget" name="Options">
     <attribute name="title">
      <string>Options</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_mailling">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>520</y>
        <width>190</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>mailling</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../C++/Projet/icons8-qr-code-60.png</normaloff>../C++/Projet/icons8-qr-code-60.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_24">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>10</y>
        <width>310</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string> Supprimer Historique</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../C++/Projet/icons8-dossier-ouvert-50.png</normaloff>../C++/Projet/icons8-dossier-ouvert-50.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-190</y>
        <width>1500</width>
        <height>1190</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../C++/backgroundDesktop.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>868</x>
        <y>-10</y>
        <width>1161</width>
        <height>821</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../C++/backgroundDesktop.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-190</y>
        <width>1511</width>
        <height>980</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../C++/backgroundDesktop.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>868</x>
        <y>-70</y>
        <width>1210</width>
        <height>830</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../C++/backgroundDesktop.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_adress_recep">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>410</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_subject">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>410</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_mailText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>410</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="help_button">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Documentation</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>90</y>
        <width>451</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_historique">
      <property name="geometry">
       <rect>
        <x>1015</x>
        <y>90</y>
        <width>441</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_on">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>560</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_of">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>620</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>label_11</zorder>
     <zorder>label_10</zorder>
     <zorder>pushButton_mailling</zorder>
     <zorder>pushButton_24</zorder>
     <zorder>lineEdit_adress_recep</zorder>
     <zorder>lineEdit_subject</zorder>
     <zorder>plainTextEdit_mailText</zorder>
     <zorder>help_button</zorder>
     <zorder>textEdit</zorder>
     <zorder>tableView_historique</zorder>
     <zorder>pushButton_on</zorder>
     <zorder>pushButton_of</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1458</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
